RPCTests/modules/async.js

```diff
@@ -0,0 +1,5289 @@
+(function (global, factory) {
```

Comment:
Please don't copy foreign code into our repository, especially not without copyright notice. Isn't there an NPM package that does this?